index.js

```diff
@@ -1,173 +1,31 @@
-require("dotenv").load();
-require(`${process.cwd()}/extenders/Guild.js`);
+require("dotenv").config();
```

Comment:
Why was this changed from `.load()` to `.config()`?

Reply:
Because `.load()` wasent loading the config.

Reply:
Some reason that's messed up, because it's working fine for Misaki in production.

Reply:
Odd